include/libopencm3/stm32/h7/timer.h

```diff
@@ -0,0 +1,42 @@
+/** @defgroup timer_defines Timer Defines
+
+@brief <b>libopencm3 Defined Constants and Types for the STM32H7xx Timers</b>
+
+@ingroup STM32H7xx_defines
+
+@version 1.0.0
+
+@author @htmlonly &copy; @endhtmlonly 2018 Karl Palsson <karlp@tweak.net.au>
+
+LGPL License Terms @ref lgpl_license
+*/
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2018 Karl Palsson <karlp@tweak.net.au>
```

Comment:
yeah, on this one I think we should just drop copyright alltogether :) there's nothing here